src/components/results/ListItem.vue

```diff
@@ -0,0 +1,37 @@
+<script setup>
+import {ref} from 'vue';
+
+const props = defineProps({
+  item: Object,
```

Comment:
This object needs to be typed, but I guess it might look better if using individual prop for each field of this object.